jinahub/indexers/dbms/PostgreSQLIndexer/postgreshandler.py

```diff
@@ -1,9 +1,17 @@
 __copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
+import psycopg2
+from jina import DocumentArray, Document
+from jina.logging.logger import JinaLogger
 from typing import Optional
 
-from jina.logging.logger import JinaLogger
+
+def doc_without_embedding(d: Document):
+    new_doc = Document(d, copy=True)
+    new_doc.ClearField('embedding')
```

Comment:
I think the preferred 2.0 API is `new_doc.pop('embedding')`. Direct handling of protobuf data types is discouraged. Can you double-check?